src/ruby/yast/y2base.rb

```diff
@@ -0,0 +1,131 @@
+module Yast
+  module Y2Base
```

Comment:
well, goal is to have module that can be tested...maybe `Y2BaseHelpers` is good enough name?